public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -292,6 +295,17 @@ export function AnomalyResults(props: AnomalyResultsProps) {
       </p>
     ) : isInitializingNormally ? (
       <p>
+        {detector.initProgress
```

Comment:
Do we want to show the message besides isInitializingNormally?  When it is over time, I can see a need for customer to ask when it is gonna finish.

Reply:
good point, will add it for overtime case

Reply:
will also add that message to missing data case as well.